### ISSUE TITLE
Items in narrative component should be in a list (WCAG 1.3.1) [QAC]

### ISSUE THREAD

**Issue by ftourangeau** [MeD-DMC/Adapt-accessibilityfixes#203]
**COURSE:**
https://esdc-edsc-adapt.canadacentral.cloudapp.azure.com/preview/5dc446fd27f08836496b6b9e/63a32079b75e454dafed1e1d/
*******************************************************************************
Page:
Lesson 3: Observation Print

Description:
Under "Overview of the Observation Print Module" there is a group of elements that navigate the slides. As these elements are related to each other they should be in a list.

Consider putting the elements in a list to better inform the user of their relationship.

Please see the attached screenshot(s) for example.

![image](https://user-images.githubusercontent.com/93537804/229619318-defa2232-bb25-49d0-aabd-02177a0ff886.png)

*******************************************************************************
